pkg/runtime/grpc/grpc-runtime.go

```diff
@@ -89,11 +116,11 @@ func New(skipInfo bool) *Runtime {
 		log.Warnf("could not store gadget info: %v", err)
 	}
 
-	return r
+	return nil
 }
 
 func (r *Runtime) UpdateDeployInfo() error {
```

Comment:
I know it was before, but shouldn't `UpdateDeployInfo` and `InitInfo` be moved to info.go?